recipe/meta.yaml

```diff
@@ -51,16 +52,21 @@ requirements:
     - requests >=2.18,<3.0
     - requests-toolbelt >=0.9.1,<0.11.0
     - shellingham >=1.5,<2.0
-    - tomli >=2.0.1,<3.0.0  # [py<311]
+    - tomli >=2.0.1,<3.0.0
```

Comment:
This should be fine because of:
https://github.com/python-poetry/poetry/blob/1.3.0/src/poetry/utils/_compat.py#L11-L15

Reply:
Ahh great, that was the only thing I wasn‘t sure of.